Model/Profile.cs

```diff
@@ -23,6 +23,7 @@ public static void Load(string profileName)
                 if((rawObject != null)){
                     profile.AHK = JsonConvert.DeserializeObject<AHK>((rawObject["AHK"]).ToString());
                     profile.Autopot = JsonConvert.DeserializeObject<Autopot>((rawObject["Autopot"]).ToString());
+                    profile.AutoBuff = JsonConvert.DeserializeObject<AutoBuff>((rawObject["AutoBuff"]).ToString());
```

Comment:
Needs
`profile.AutoRefresh = JsonConvert.DeserializeObject<AutoBuff>((rawObject["AutoRefresh"]).ToString());` to save AutoRefresh Key in profile.